AuthenticatorShared/UI/Platform/Application/AppCoordinator.swift

```diff
@@ -63,8 +63,12 @@ class AppCoordinator: Coordinator, HasRootNavigator {
     func handleEvent(_ event: AppEvent, context: AnyObject?) async {
         switch event {
         case .didStart:
+            let accountId = await services.stateService.getActiveAccountId()
+            let hasTimeout = services.appSettingsStore.vaultTimeout(userId: accountId) !=
```

Comment:
🤔 Could we get this setting from the state service as well? In PM, at least, we have convenience methods there that avoid needing to also get the account ID.

Reply:
Yeah that makes sense. It is awkward to have to do both.

Reply:
Should be cleaned up a bit in the latest commit. 👍